src/quantum/config.js

```diff
@@ -51,75 +52,159 @@ const PLATFORMS = [
     bits: 32,
     os: 'win',
     label: 'Firefox',
-    framework: 10,
-    platform: 'windows7-32',
-    option: 'pgo',
-    project: 'mozilla-central',
+    platform: "win32",
+    seriesConfig: {
+      and: [
+        {
+          or: [
+            {
+              eq: {
+                platform: 'windows7-32',
+                options: "pgo"
+              }
+            },
+            {
+              eq: {
+                platform: 'windows7-32-shippable',
+                options: "opt"
+
+              }
+            }
+
+          ]
+        },
+        {
+          eq: {
+            framework: 10,
+            repo: 'mozilla-central',
+          }
+        },
+
+      ]
+    }
```

Comment:
Iterating over `seriesConfig` is a bit complicated with all of the nesting. If the issue is about properly being able to filter through the data, have we thought about using another tool? For example, there's https://github.com/crcn/sift.js which exposes many operators like `$eq`, `$and`, and `$or` which could be used to filter through the array.